.semaphore/semaphore.yml

```diff
@@ -0,0 +1,92 @@
+---
+version: "v1.0"
+name: "Vaultenv CI Pipeline"
```

Comment:
Nit: the CI pipeline is called `Vaultenv CI Pipeline`. I can guess why this is the cause, but we probably do want to change it.

Reply:
oops